packages/forma-36-react-components/src/components/Dropdown/README.mdx

```diff
@@ -39,7 +39,7 @@ function DropdownExample() {
         <Button
```

Comment:
I guess we can remove Dropdown component. I think I saw it in Alex PR, but maybe it's not merged yet?

Reply:
Done